Library/Homebrew/dev-cmd/pull.rb

```diff
@@ -253,11 +259,18 @@ def pull
     verify_bintray_published(bintray_published_formulae)
   end
 
+  def test_bot_user(user)
+    test_bot = ARGV.value "test-bot-user"
+    return test_bot if test_bot
+    return "BrewTestBot" if user.casecmp("homebrew").zero?
```

Comment:
Not sure I understand `user.casecmp("homebrew").zero?`; what's it doing?

Reply:
If the GitHub organization that the PR is submitted to is `Homebrew` (case insensitive comparison), then the GitHub test-bot user is `BrewTestBot`.

Reply:
Definitely need a case-insensitive comparison here? If so, `user.downcase == "homebrew"` is more typical style.